CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+### AMI Release v20230509
+* amazon-eks-gpu-node-1.26-v20230509
+* amazon-eks-gpu-node-1.25-v20230509
+* amazon-eks-gpu-node-1.24-v20230509
+* amazon-eks-gpu-node-1.23-v20230509
+* amazon-eks-gpu-node-1.22-v20230509
+* amazon-eks-arm64-node-1.26-v20230509
+* amazon-eks-arm64-node-1.25-v20230509
+* amazon-eks-arm64-node-1.24-v20230509
+* amazon-eks-arm64-node-1.23-v20230509
+* amazon-eks-arm64-node-1.22-v20230509
+* amazon-eks-node-1.26-v20230509
+* amazon-eks-node-1.25-v20230509
+* amazon-eks-node-1.24-v20230509
+* amazon-eks-node-1.23-v20230509
+* amazon-eks-node-1.22-v20230509
+
+[Release versions](https://docs.aws.amazon.com/eks/latest/userguide/eks-linux-ami-versions.html) for these AMIs:
+* `1.26.2-20230509`
+* `1.25.7-20230509`
+* `1.24.11-20230509`
+* `1.23.17-20230509`
+* `1.22.17-20230509`
+
+Binaries used to build these AMIs are published:
+* s3://amazon-eks/1.26.2/2023-03-17/
+* s3://amazon-eks/1.25.7/2023-03-17/
+* s3://amazon-eks/1.24.11/2023-03-17/
+* s3://amazon-eks/1.23.17/2023-03-17/
+* s3://amazon-eks/1.22.17/2023-03-17/
+
+AMI details:
+* `kernel`:
+  * Kubernetes 1.23 and below: 5.4.241-150.347.amzn2
+  * Kubernetes 1.24 and above: 5.10.178-162.673.amzn2
+* `dockerd`: 20.10.23-1.amzn2.0.1
+  * **Note** that Docker is not installed on AMI's with Kubernetes 1.25+.
+* `containerd`: 1.6.19-1.amzn2.0.1
+* `runc`: 1.1.4-1.amzn2
+* `cuda`: 11.4.0-1
+* `nvidia-container-runtime-hook`: 1.4.0-1.amzn2
+* `amazon-ssm-agent`: 3.1.1732.0-1.amzn2
+
+Notable changes:
+- The new AMIs have updated docker version 20.10.23-1.amzn2.0.1 that addresses two docker CVEs; CVE-2022-36109 - docker  and CVE-2022-37708 - docker.
```

Comment:
We need links for these CVE's, use ALAS: https://alas.aws.amazon.com/AL2/ALASDOCKER-2023-022.html

Reply:
Addressed in the new commit.